tactics/class_tactics.ml

```diff
@@ -1373,22 +1373,22 @@ module Search = struct
          else raise Not_found
        with Logic_monad.TacticFailure _ -> raise Not_found
 
-  let evars_eauto depth only_classes unique dep st hints p evd =
+  let evars_eauto depth only_classes unique dep st hints p evd env =
```

Comment:
Same here, `env` should be before `evd  and ideally the first arguments of the function.